Event/EventBase.gd

```diff
@@ -1,13 +1,13 @@
 extends Resource
 class_name EventBase
 
-const EVENTS_CLASSIFICATION: Array[EventBase] = [EventMob.new(""), EventRandom.new(""), EventShop.new(""), EventHeal.new("")]
+var EVENTS_CLASSIFICATION: Array[EventBase] = [EventMob.new(), EventRandom.new(), EventShop.new(), EventHeal.new()]
```

Comment:
i'm not sure it's needed to have a .new() here i think, it's not really useful (will will create new or copy later on if we need it anyway

Reply:
Updated!